Support for @pact-foundation/pact v13

### Checklist

Before making a feature request, I have:

- [x] [Searched the issues to check that this feature hasn't been requested before](https://github.com/pact-foundation/pact-js-core/issues?q=is%3Aissue)
- [x] Checked the documentation to see if it is possible to do what I want already

### Feature description

Support for the new major version (v13) of @pact-foundation/pact.

### Use case

Pact files are generated in my project using jest-pact, and I would like to update @pact-foundation/pact to it's latest version.

Error after bumping up @pact-foundation/pact to v13:
```
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: jest-pact@0.11.0
npm error Found: @pact-foundation/pact@13.0.0
npm error node_modules/@pact-foundation/pact
npm error   dev @pact-foundation/pact@"^13.0.0" from the root project
```

--- a/package.json
+++ b/package.json
@@ -43,7 +43,7 @@
   "devDependencies": {
     "@commitlint/cli": "^17.6.7",
     "@commitlint/config-conventional": "^19.2.2",
-    "@pact-foundation/pact": "^12.5.0",
+    "@pact-foundation/pact": "^13.0.0",
     "@pact-foundation/pact-js-prettier-config": "^1.0.0",
     "@tsconfig/node14": "^1.0.3",
     "@types/jest": "^29.2.3",
@@ -72,7 +72,7 @@
     "typescript": "4.6.4"
   },
   "peerDependencies": {
-    "@pact-foundation/pact": "^v10.0.0-beta.61 || ^10.0.2 || ^11.0.2 || ^12.0.0",
+    "@pact-foundation/pact": "^v10.0.0-beta.61 || ^10.0.2 || ^11.0.2 || ^12.0.0 || ^13.0.0",
     "jest": "^24.0.0 || ^25.0.0 || ^26.0.0 || ^27.0.0 || ^28.0.0 || ^29.0.0"
   },
   "lint-staged": {

